guides/common/modules/proc_adding-kubevirt-connection.adoc

```diff
@@ -32,7 +32,7 @@ Make a note of this token to use later in this procedure.
 . In the *Name* field, enter a name for the new compute resource.
 . From the *Provider* list, select *{KubeVirt}*.
 . In the *Description* field, enter a description for the compute resource.
-. In the *Hostname* field, enter the address of the {KubeVirt} server that you want to use.
+. In the *Hostname* field, enter the FQDN, hostname, or IP address of the {KubeVirt} server that you want to use.
```

Comment:
Isn't hostname and FQDN pretty much the same here?

It does sound like the field is poorly named and `Host` would have been a better label.

Reply:
Depending on the user's DNS configuration, it can be different.

If you change the label, please let me know.

Reply:
The KubeVirt plugin is a bit unmaintained at the moment so I doubt it'll be changed soon.

Reply:
Marek Hulan told me that it might even be removed in a future release. So I wouldn't consider it an issue.